content/rs/administering/designing-production/performance/shard-placement.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: Configuring Shard Placement
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+{{< embed-md "shard-placement-intro.md"  >}}
+
+## Default Shard Placement Policy
+
+The cluster configuration includes a default policy for the shard placement policy.
+When you create a database, this default policy is applied to the new database.
+The initial policy for all new clusters is `dense`.
+
+You can see the current default shard placement policy in `rladmin info cluster`:
+
+```src
+$ rladmin info cluster
+cluster configuration:
+   repl_diskless: enabled
+   default_non_sharded_proxy_policy: single
+   default_sharded_proxy_policy: single
+   default_shards_placement: dense
```

Comment:
I'd make the relevant parts of the output bold for easier reading

Reply:
Changed to image